_docs/guides/endpoints.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Install Istio for Google Cloud Endpoints Services  
```

Comment:
This probably should be a task, not a guide. The definition of a guide page is:

A guide page describes a fully working stand-alone example highlighting a particular set of features. Guides must have easy to follow setup and usage instructions so users can quickly run the sample themselves and experiment with changing the sample to explore the system.